WordPress/Classes/ViewRelated/Views/List/NotificationsList/AvatarView.swift

```diff
@@ -79,7 +82,7 @@ struct AvatarView<S: Shape>: View {
 
     private func avatar(url: URL?) -> some View {
         let processedURL: URL?
-        let size = Int(ceil(diameter * UIScreen.main.scale))
+        let size = Int(ceil(diameter * screenScale))
```

Comment:
This isn't related to the PR but I came across it during this PR and figured I'd address it.